ChiModules/LinearBoltzmannSolver/GroupSet/lbs_groupset.cc

```diff
@@ -15,7 +15,7 @@ extern ChiMPI& chi_mpi;
 LBSGroupset::LBSGroupset()
 {
   quadrature = nullptr;
-  iterative_method = NPT_GMRES;
+  iterative_method = LinearBoltzmann::IterativeMethod::NONE;
```

Comment:
should there be something else by default ??

Reply:
No preferences. What would you like to see?